### ISSUE TITLE
Languages Feature

### ISSUE THREAD

**Issue by Cooolaid** [adobe/brackets#11637]
Its really annoying to have to open a new document every single time and then save document every single time to get the "Preferred" language I want to program in.

I would like this to be changed. I would like to just click the (file type) and select my preferred language I want in the bottom right corner or in the Brackets Settings.
without having to open a new document and then save the document to get this.

I also would like to be able to have the brackets software to open in "MY" Preferred language I always want it to open to. So every time I open it, it opens in the language I want / or selected in the settings of brackets to always open to.

I know this is a bit hard to understand, I am doing my best to explain it in plain text.
which for me its a bit hard to explain. sorry. 
But this is one of those things thats extremely annoying while using Brackets software.
and would like the creators of this program to look into this.
thank you.

**Comment by ryanstewart** [comment]
Yup, this is annoying and is something we have on the backlog - https://trello.com/c/CNXuU4TY/1369-switch-language-type-of-untitled-documents - I don't have an ETA, but we know we need to have it.
